app/src/main/res/values/strings.xml

```diff
@@ -158,12 +159,15 @@
     <string name="fingerprint_scan_continue">Scan your Fingerprint to continue</string>
     <string name="fingerprint_scan_unlock">Unlock the app with biometrics</string>
     <string name="fingerprint_touch_sensor">Touch Sensor</string>
+    <string name="fingerprint_forgot">Forgot?</string>
```

Comment:
how is possible to forget fingerprint? that assumption i did while reading key name.